talent/sourcerers/gomesfernanda.md

```diff
@@ -0,0 +1,58 @@
+# Fernanda Gomes
+
+**General**
+- What's your name? How do your friends call you? 
+  - Fernanda. My friends call me "Fê" or "Fer".
+- When were you born?
+  - June 2nd, 1984.
+- Where are you from? Where have you lived?
+  - I'm naturally from Santos/BR. I lived in Santos, Campinas and São Paulo (Brazil, the three of them).
+- What languages (natural and programming ones) do you know?
+  - Portuguese, English, Spanish.
+  - Python, Go; learnt Pascal and C++ in college but forgot.
+- Would like to share your Twitter, Instagram or Medium or any other social media account?
+  - [LinkedIn](https://www.linkedin.com/in/g0mesfernanda/) | [Twitter](http://twitter.com/gofernanda_) | [Instagram](http://www.instagram.com/ohhrosie) | [Last.fm](https://www.last.fm/pt/user/fergomes)
+
+**Work**
+- What are you going to be doing at source{d}?
+  - Data Intelligence team, really close to Products team. I work collecting and turning data into nice dashboards, reports, charts, according to requests from Operations, Products, Developer Community, Engineering.
```

Comment:
Better phrased as:

- Data Intelligence, under Product Team. I work turning the internal and external reality into actionable insights for decision-making in a variety of levels and teams; this could take the form of spreadsheets, data pipelines, software, dashboards, reports, presentations, recommendations… a.k.a. data science when closer to business than to research.

Reply:
@gomesfernanda you were selling yourself short ;)

Reply:
PR with this adjustment! https://github.com/src-d/guide/pull/181
thank you very much